### PR TITLE
[3829][15.0][UPD] dotfiles

### PR DESCRIPTION
**Pull request by liuhehe1995**
[3829](https://www.quartile.co/web#id=3829&menu_id=505&cids=3&action=1457&model=project.task&view_type=form)

### CODE REVIEW

File: website_odoo_debranding/templates/disable_odoo.xml [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
         priority="20"
     >
-        <xpath expr="//div[hasclass('o_brand_promotion')]" position="replace">
-            <div class="o_brand_promotion" />
-        </xpath>
-    </template>
+      <xpath expr="//div[hasclass('o_brand_promotion')]" position="attributes">
+          <attribute name="class">o_brand_promotion d-none</attribute>
+      </xpath>
+  </template>
```
Comment (AungKoKoLin1997) [inline website_odoo_debranding/templates/disable_odoo.xml]: Why did these changes? Is there any reason not to follow the adjustments from OCA?
https://github.com/OCA/website/commit/ea8f7775093b0b65d67e3037150c236cf5c8cfa7

File: web_m2x_options/__init__.py [hunk 2]
```diff
@@ -1,2 +1,3 @@
 from . import models
-from . import tests
+
+# from . import tests
```
Comment (AungKoKoLin1997) [inline web_m2x_options/__init__.py]: Just remove it.

**Review by AungKoKoLin1997 — CHANGES_REQUESTED** [review]

File: website_odoo_debranding/templates/disable_odoo.xml (hunk 1 above)
Comment (liuhehe1995) [reply website_odoo_debranding/templates/disable_odoo.xml]: When I run `pre-commit run --all-files`, there is a prompt asking me to change this line. （template version 1.18）

`website_odoo_debranding/templates/disable_odoo.xml:11 Dangerous use of "replace" from view with priority `20 < 99`. Only replace as a last resort. Try position="attributes", position="move" or t-if="False" first - [xml-dangerous-qweb-replace-low-priority]`

image

File: web_m2x_options/__init__.py (hunk 2 above)
Comment (liuhehe1995) [reply web_m2x_options/__init__.py]: Thank you,Removed.

File: website_odoo_debranding/templates/disable_odoo.xml (hunk 1 above)
Comment (liuhehe1995) [reply website_odoo_debranding/templates/disable_odoo.xml]: I understood. 
I should change line 9 `priority="20"` to `priority="999"`

Comment (liuhehe1995) [reply website_odoo_debranding/templates/disable_odoo.xml]: I made a mistake. The priority should be 99. not 999.
Created a new Commit to fix it.

File: .copier-answers.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+ci: GitHub
+generate_requirements_txt: true
+github_check_license: true
+github_ci_extra_env: {}
+github_enable_codecov: false
+github_enable_makepot: false
+github_enable_stale_action: false
+github_enforce_dev_status_compatibility: false
+include_wkhtmltopdf: false
+odoo_test_flavor: Both
```
Comment (AungKoKoLin1997) [inline .copier-answers.yml]: Please adjust to be the same with other xxx-oca repos

**Review by AungKoKoLin1997 — COMMENTED** [review]
@liuhehe1995  Please run `copier update` again. The dotfiles are not latest anymore.

**Review by AungKoKoLin1997 — APPROVED** [review]

**Review by yostashiro — APPROVED** [review]

**Pull request merged**